chapter_10/10-12.py

```diff
@@ -0,0 +1,17 @@
+import json
+from pathlib import Path
+
+# does everything the same but uses the name to find the correct file so you can
+# store multiple people
+name = input("What is your name? ")
+path = Path(f"{name}.json")
+if path.exists():
+    contents = path.read_text()
+    favorite = json.loads(contents)
+    print(f"Welcome back {name}")
```

Comment:
```suggestion
    print(f"Welcome back, {name}.")
```

Add comma/period for grammar reasons